lib/helper.rb

```diff
@@ -0,0 +1,55 @@
+module Helper
+  def self.shape
```

Comment:
You have a lot of methods in this module that are using `print`, `puts`, or `chomp` and that's against the [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/main_rb.png). Please fix that. TIP: you can return a string and then print it on main.rb if you must.